pkg/services/folder/folderimpl/folder_test.go

```diff
@@ -374,16 +382,6 @@ func TestIntegrationFolderService(t *testing.T) {
 				require.NoError(t, err)
 			})
 
-			t.Run("When get folder by title should return folder", func(t *testing.T) {
```

Comment:
could we add for "Get", passing the title as query parameter? Would that make sense/add value for the test suite?

Reply:
oh! I saw that you already added `TestIntegrationGetDashFolderStore` above!